components/calendar/calendar.js

```diff
@@ -363,16 +365,19 @@ class Calendar extends LocalizeStaticMixin(LitElement) {
 			return html`<tr>${weekHtml}</tr>`;
 		});
 		const activeDate = `${this._tableInfoId}-${this._focusDate.getFullYear()}-${this._focusDate.getMonth()}-${this._focusDate.getDate()}`;
+		const labelId = `${this._tableInfoId}-heading`;
+		const labelledBy = this.dialog ? labelId : undefined;
 		const heading = `${calendarData.descriptor.calendar.months.long[this._shownMonth]} ${this._shownYear}`;
+		const role = this.dialog ? 'dialog' : undefined;
 		return html`
-			<div class="d2l-calendar">
+			<div aria-labelledby="${ifDefined(labelledBy)}" aria-modal="${ifDefined(this.dialog)}" class="d2l-calendar" role="${ifDefined(role)}">
```

Comment:
Note: I tried using `aria-modal` with the `d2l-dialog` components and I had mixed results.  In fact, I think it might have caused some problems.  I had to exclude it.  Something to watch out for.

Reply:
Discussed with Dave and decided that it was best to remove `aria-modal` since it isn't really applicable here and could cause problems when calendar is used with VoiceOver